GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -576,6 +574,17 @@ protected override void OnLoad(EventArgs e)
             _formBrowseDiagnosticsReporter.Report();
 
             base.OnLoad(e);
+
+            RestoreLayout(); // a second time in order to really restore the splitter positions
```

Comment:
What happens if we remove the first one?

Reply:
> What happens if we remove the first one?

Seems to work. One might see moving the splitters more than without.
With `SetSplitterPositions` called last, also the CommitInfo width is restored.
(I feared to break the DPI scaling or even the translation - lots of nested functions. I will not test those myself.)